### DISCUSSION TITLE
Release 0.1.3 Question

### DISCUSSION THREAD

**Discussion by josephvtesta** [RobertD502/home-assistant-flair#38] [Q&A] [answered]
Reading the changelog, it says "The Select entity Structure mode and Number entity Home set point have been removed. These two have been combined into a Climate entity called Structure. NOTE: If your system mode is set to manual, this entity will become unavailable. In addition, the target temperature (Previously known as Home set point) can only be changed if the Set point controller entity is set to Flair app. It is safe to delete the old Structure mode and Home set point entities."

Does this mean pucks are no longer usable via home assistant when flair is in manual mode? I upgraded to verify my understanding it did break the pucks and my ability to set specific temps in each room a puck is in as puck climate entities show as "unavailable".

**Comment by RobertD502** [comment, marked as answer]
Could you explain a little further as Pucks don't have a climate entity in Home Assistant. See below:

![image](https://user-images.githubusercontent.com/52541649/194593563-bb855fce-449d-415d-bdd4-dc73d2b9893a.png)

**Reply by RobertD502** [reply]
If you are referring to climate entities for "Rooms" that are unavailable in manual mode, changing the set point isn't available on those in the flair app - this is why the climate entity for Rooms becomes unavailable when in "Manual" mode:
![IMG_5148](https://user-images.githubusercontent.com/52541649/194617368-6d4d809f-02f7-4f03-9126-e05ac6c508d9.jpg)


Climate entities for "Rooms" were previously available while in manual mode and you could change the set point for them, but it didn't actually do anything because Flair servers didn't care since you can't set individual room temps when in "manual" mode as seen in the image above.


If you're referring to the new climate entity that is created for Flair structures, this follows the same concept that is seen in the Flair app - setting a Home set point is not available when in manual mode (set point controller options aren't even available):


![IMG_5149](https://user-images.githubusercontent.com/52541649/194618330-37933209-870e-420e-a875-4bc53e3ee9d9.jpg)

![IMG_5150](https://user-images.githubusercontent.com/52541649/194618345-d19c583d-2a9c-4d92-b27b-fe762c35578f.jpg)
